### PR TITLE
Update capture path

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/services/data/configure-data-capture.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ To capture data from one or more robots, you must first add the [Data Management
       "type": "data_manager",
       "attributes": {
         "sync_interval_mins": 1,
-        "capture_dir": "/.viam/capture"
+        "capture_dir": "~/.viam/capture"
```
[the remark this answers is not in the document]
Comment (npentrel) [reply docs/services/data/configure-data-capture.md:42]: I didn't realize it doesn't work right now

File: docs/services/data/configure-cloud-sync.md [hunk 2]
```diff
@@ -72,7 +72,7 @@ In the example pictured here, the Data Management Service syncs the configured c
       "type": "data_manager",
       "attributes": {
         "sync_interval_mins": 5,
-        "capture_dir": "/.viam/capture",
+        "capture_dir": "~/.viam/capture",
```
Comment (agreenb) [inline docs/services/data/configure-cloud-sync.md:75]: Can the capture_dir in this file be updated too?

File: docs/services/data/configure-cloud-sync.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ To change the sync interval, specify an interval in minutes in the interval fiel
       "type": "data_manager",
       "attributes": {
         "sync_interval_mins": 0.1,
-        "capture_dir": "/.viam/capture"
+        "capture_dir": "~/.viam/capture"
```
Comment (agreenb) [inline docs/services/data/configure-cloud-sync.md:33]: Here too

**Review by agreenb — APPROVED** [review]
LGTM after the two small changes, thank you!

File: docs/services/data/configure-cloud-sync.md (hunk 3 above)
Comment (npentrel) [reply docs/services/data/configure-cloud-sync.md:33]:
```suggestion
        "capture_dir": ""
```

File: docs/services/data/configure-cloud-sync.md (hunk 2 above)
Comment (npentrel) [reply docs/services/data/configure-cloud-sync.md:75]:
```suggestion
        "capture_dir": "",
```

**Pull request merged**